### ISSUE TITLE
Mark forbidden paths/highways more clearly as such

### ISSUE THREAD

**Issue by polarbearing** [gravitystorm/openstreetmap-carto#4465]
Currently, we render paths/highways with access restrictions by reducing their visibility and use pale fill marks.

We should discuss if this is a good strategy. With the significant increase of the usage of OSM data, in particular in outdoor apps, there are a lot of complaints from land owners about hiker/biker etc trespassing their land, because the app users just see paths and assume free access. Land owners often request the deletion of such paths, which the DWG typically declines, respecting the ground truth, while they of course need to be tagged accordingly. 

More visibly marking such forbidden paths would help in the arguments with such landowners that OSM did their best to do so, and provide an example for the rendering styles in outdoor maps/routers, which we cannot influence directly.

**Comment by imagico** [comment]
I don't think this suggestion is compatible with [our goals](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md).

There is no argument provided why the suggested change (making paths the use of which displeases land owners more visible rather than less) would be of benefit to anyone.  I have never seen any map which follows a strategy of displaying paths that are of limited use or which are discouraged or impossible to be used more prominently than normal paths.  Also keep in mind that in OSM we map what is verifiable on the ground.  The desires of land owners are not verifiable.  And the typical case of land owners complaining about paths on their land being shown in OSM is exactly when there is a discrepancy between the desires of the land owner and the situation on the ground.  Like when the land owner would like to physically prevent access to their land (like with a fence and gate) but practically can't - hence they try to virtually build that gate in OSM.  Even if we wanted to do that in this map style that would not be a good idea because we would encourage mappers to map something that is not to be mapped in OSM according to the project's core principles (that is the subjective choice to advise people in the area not to use a certain path).

Independent of all of that even if we wanted to follow that suggestion it hardly seems practically feasible.  We currently have several issues about path rendering being hard to real (#1748, #1765, #1793) - fixing that and differentiating additional classes of paths more prominently is not likely going to be possible without completely messing up map readability.

**Comment by SomeoneElseOSM** [comment]
To be honest, I think that OSM Carto is actually one of the better map styles when it comes to showing "what is private and what isn't".  Historically, I suspect that most of the time when the DWG has had complaints of this sort it's been about some other app using it's own rendering**.  OSM Carto isn't perfect, but as a general map style it would be really difficult to show "you're allowed to walk down here, but only cycle under certain circumstances, and you're not allowed to drive".

> More visibly marking such forbidden paths would ... provide an example for the rendering styles in outdoor maps/routers, which we cannot influence directly.

As @imagico says path rendering has been a problem historically, and I'm on record as saying that OSM Carto is too urban-centric in this regard, and I can completely understand why a hiking app would want to roll their own rendering rather than use a style based on this one.  Unfortunately the app developers often don't understand the depth of tagging information that OSM data contains (including access tags) and just show "there is a path here", not whether it would be legally or physically appropriate to use it.  I wouldn't want to use a hiking map based on this style, because it misses most of the information I'd need at any useful scale; and I wouldn't want to use a hiking app with its own map style created by someone who barely knew what OSM was and what data it contained.

This whole area is a difficult problem partly because the goals described in [CARTOGRAPHY.md](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md) are to some extent contradictory - a map can't be an "important feedback mechanism" if it only shows a small percentage of tags, and it can't be an "exemplar stylesheet" if it shows absolutely everything.  For more info, compare the number of tags supported by map styles at [taginfo projects](https://taginfo.openstreetmap.org/projects) compared with those supported by editors and the Name Suggestion Index.

** a couple of culprits are regular offenders for various reasons but I'll spare their blushes here.

**Comment by jeisenbe** [comment]
This issue is similar to #2561 which was noticed after #2257 and closed. In that PR access markings were changed to be less prominent, since making no access or private access roads visually prominent was not helpful.  

As mentioned there, "improvements are always possible, and if someone wants to play around with colour, width, spacing, etc and come up with something they feel is better, they can open a pull request.”

@polarbearing  do you have any ideas about how we could more clearly show that certain paths or roads are private, while not unintentionally emphasizing those ways, or causing a confusing rendering?

**Comment by zekefarwell** [comment]
Although I understand that this suggestion may not be compatible with the goals of this project, I also understand that this style is being used by many people in ways that are outside the scope of those goals because of the liberal tile usage policy.  As it is the standard layer on osm.org, this style is also viewed in the OSM ecosystem as a de-facto reference implementation.  With this in mind I feel displaying access information as clearly as possible is the responsible thing to do. I'd suggest something similar to what [GaiaGPS](https://www.gaiagps.com/map) has done in their default Gaia Topo map layer.

When tagged as `access=no` trails get "No Access" appended to the name:
![gaia-topo-no-access-example](https://user-images.githubusercontent.com/281482/134780478-a9c23cd2-1b0e-4f1a-9970-35999f0edcd2.png)

when tagged `access=private` they get "Private" appended:
![gaia-topo-private-example](https://user-images.githubusercontent.com/281482/134780477-1d28dd95-8977-48e0-bac5-2b8f3b47cc6c.png)

This is also clearly presented in the legend:
![gaia-topo-legend](https://user-images.githubusercontent.com/281482/134780471-28d38e39-7687-4ffb-97c6-a1480876e364.png)

By comparison, the OSM standard layer legend doesn't show how private or no access trails are rendered.  I'm not sure if that's handled in this repo, or in [openstreetmap-website](https://github.com/openstreetmap/openstreetmap-website).
![osm-standard-layer-legend](https://user-images.githubusercontent.com/281482/134780473-18554459-6175-453e-a6d7-fb320d6ca75d.png)

**Comment by zekefarwell** [comment]
Also just to clarify, simply appending the English text "no access" or "private" as Gaia does wouldn't work for non-English speaking parts of the world.  The text could be rendered in the primary local language just like names currently are, but this would require a large list of these localized text strings.  A simpler approach would be to usage a universal symbol such as the no-entry ⛔ or prohibited 🚫 symbols.

https://emojipedia.org/no-entry/
https://emojipedia.org/prohibited/

**Comment by imagico** [comment]
Yes, using pictorial symbols to illustrate access restrictions is in principle a possible approach.  The difficulty here - apart from Mapniks limitations in placing symbols - is that this has a high likelihood to be confused with mapped point features.  Like map users seeing such a symbol on a path and likely assuming there is a 'no trespassing' sign at the specific location the symbol is rendered at.  And that assumption makes a lot of sense since a point symbol is typically used to indicate something that applies to a specific location and not to characterize a line that happens to run through that point.

That does not mean the very idea is categorically not suitable but this is not something that would be easy to make work, especially not with the wide range of road/path line signatures we use.

**Comment by zekefarwell** [comment]
I was imagining using the symbol inline with the name so it wouldn't read as a point feature.  Something like this:

![Screen Shot 2021-09-27 at 4 57 00 PM](https://user-images.githubusercontent.com/281482/134985354-013c9c1e-4098-4f01-9311-bb3521099abf.png)

Of course if there is no name, then the symbol alone would look much more like a  point feature.  Perhaps if the symbol were centered across the line and used a similar color to the line that might work better:

![Screen Shot 2021-09-27 at 5 04 08 PM](https://user-images.githubusercontent.com/281482/134985991-2e676eed-2111-4f6f-8eb2-ede509d0bfe9.png)

Anyway, these are just some ideas.  Really I think just showing forbidden paths more clearly in the legend (key) would be a big improvement.

**Comment by HolgerJeromin** [comment]
> Of course if there is no name, then the symbol alone would look much more like a point feature. 

correct. Sadly probably most/many foot/cycleway with access!=yes have probably no name... 8-(

**Comment by rskedgell** [comment]
Rendering highway=footway + foot=private|no in the same style (light grey) as access=private|no rather than in red might help a little here. This is possibly more of a problem because of validators which complain about access=* tags as being "too permissive" on highway=footway, highway=steps, etc.

**Comment by imagico** [comment]
That is part of #214 - which would really be nice to be addressed, considering it is our second oldest issue...
